src/common/components/repository-row/dropdowns/edit-config-dropdown.js

```diff
@@ -0,0 +1,40 @@
+import React, { PropTypes } from 'react';
+import url from 'url';
+
+import { parseGitHubRepoUrl } from '../../../helpers/github-url';
+import { Row, Data, Dropdown } from '../../vanilla/table-interactive';
+
+const getTemplateUrl = (repositoryUrl, configFilePath) => {
+  const { fullName } = parseGitHubRepoUrl(repositoryUrl);
+  const templateUrl = url.format({
+    protocol: 'https:',
+    host: 'github.com',
+    pathname: `${fullName}/edit/master/${configFilePath}`
+  });
+
+  return templateUrl;
+};
+
+const EditConfigDropdown = ({ repositoryUrl, configFilePath }) => {
+  const templateUrl = getTemplateUrl(repositoryUrl, configFilePath);
+
+  return (
+    <Dropdown>
+      <Row>
+        <Data col="100">
+          <p>
+            This repo has a snapcraft.yaml file.
+            <a href={ templateUrl } target="_blank"> You can edit the file on GitHub.</a>
```

Comment:
Also space in link gets underlined:
<img width="299" alt="screen shot 2017-03-16 at 16 49 55" src="https://cloud.githubusercontent.com/assets/83575/24005248/c5922022-0a68-11e7-93d8-4986ea76e323.png">